soh/soh/SohGui.cpp

```diff
@@ -43,7 +43,7 @@
 #include "Enhancements/resolution-editor/ResolutionEditor.h"
 #include "Enhancements/debugger/MessageViewer.h"
 
-bool ToggleAltAssetsAtEndOfFrame = false;
+bool prevAltAssets = false;
```

Comment:
Not sure why this was ever in here, but I think it should be dropped. This var should only be used/set withing OTRGlobals.cpp

Reply:
It was there because it was used in the menu toggle CVarSet cancel thingy. Was probably added here first, then externed into OTRGlobals later.